.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+# https://medium.com/@gottfrois/github-actions-for-elixir-phoenix-app-with-cache-6ca33f628459
+
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+env:
+  LC_CTYPE: 'en_US.UTF-8'
+  LANGUAGE: 'en_US.UTF-8'
+  MIX_ENV: test
+  # One of https://github.com/spesmilo/electrum/blob/master/electrum/servers.json
+  LB_ELECTRUM_CLIENT_IP: 'blackie.c3-soft.com'
```

Comment:
This seems to be adding a dependency to a specific server.

What happens when it goes down?

Reply:
You're right - relaying on a third party server is  not ideal. However I still want to keep integration test because contract between ElectrumClient and BitcoinAccointing is quite loose: it's not based on well-defined structs with at least known fields. 
And fixing that is a not a quick change from what I saw in dialyzer output. 

I pushed new change that marked integration tests as optional - by default they will be excluded in the `mix test` command.  

I can move them into the `test/integration` and add mock-based tests instead.  How does that sound?